integration/exec/eth_node.go

```diff
@@ -0,0 +1,26 @@
+package exec
```

Comment:
Why is this package called `exec`?

Reply:
It handles the life cycle of the node, executing/ starting stopping.